recipes/etcd_cluster.rb

```diff
@@ -165,6 +165,13 @@
     }
   end
 
+  directory "#{node['cookbook-openshift3']['etcd_data_dir']}member" do
```

Comment:
I put that logic to clear a pre-existing etcd member. But you are right it should not be there...